### PR TITLE
Update prerequisites

### PR DESCRIPTION
**Pull request by 5abeel**

### CODE REVIEW

File: docs/guides/setup/es2k-setup-guide.md [hunk 1]
```diff
@@ -9,17 +9,27 @@ For the ACC, see [Building for the ES2K ACC](/guides/building-for-es2k-acc).
 
 ## Prerequisites
 
+It is assumed that the ES2K IPU SDE dependencies have been installed on the
+system. The list of SDE's dependencies can be found
```
Comment (ffoulkes) [inline docs/guides/setup/es2k-setup-guide.md:13]:
```suggestion
system. The list of these dependencies can be found
```

File: docs/guides/setup/es2k-setup-guide.md [hunk 2]
```diff
@@ -9,17 +9,27 @@ For the ACC, see [Building for the ES2K ACC](/guides/building-for-es2k-acc).
...
+in `sde/tools/setup/install_dep.py` file.
+
+Prerequisites and dependencies from stratum-deps are assumed to be
```
Comment (ffoulkes) [inline docs/guides/setup/es2k-setup-guide.md:16]:
```suggestion
Prerequisites for stratum-deps are assumed to be installed.
```

File: docs/guides/setup/dpdk-setup-guide.md [hunk 3]
```diff
@@ -5,17 +5,27 @@ for the DPDK target.
 
 ## Prerequisites
 
+It is assumed that the DPDK SDE dependencies have been installed on the
+system. The list of SDE's dependencies can be found
```
Comment (ffoulkes) [inline docs/guides/setup/dpdk-setup-guide.md:9]:
```suggestion
system. The list of these dependencies can be found
```

File: docs/guides/setup/dpdk-setup-guide.md [hunk 4]
```diff
@@ -5,17 +5,27 @@ for the DPDK target.
...
+in `sde/tools/setup/install_dep.py` file.
+
+Prerequisites and dependencies from stratum-deps are assumed to be
```
Comment (ffoulkes) [inline docs/guides/setup/dpdk-setup-guide.md:12]:
```suggestion
Prerequisites for stratum-deps are assumed to be installed.
```

File: docs/guides/setup/es2k-setup-guide.md [hunk 5]
```diff
@@ -9,17 +9,27 @@ For the ACC, see [Building for the ES2K ACC](/guides/building-for-es2k-acc).
...
+installed, since some of those dependencies also apply to this
```
Comment (ffoulkes) [inline docs/guides/setup/es2k-setup-guide.md:17]:
```suggestion
Some of them also also apply to networking-recipe.
```

File: docs/guides/setup/dpdk-setup-guide.md [hunk 6]
```diff
@@ -5,17 +5,27 @@ for the DPDK target.
...
+installed, since some of those dependencies also apply to this
```
Comment (ffoulkes) [inline docs/guides/setup/dpdk-setup-guide.md:13]:
```suggestion
Some of them also apply to networking-recipe.
```

File: docs/guides/setup/dpdk-setup-guide.md [hunk 7]
```diff
@@ -5,17 +5,27 @@ for the DPDK target.
...
+module, networking-recipe.
```
Comment (ffoulkes) [inline docs/guides/setup/dpdk-setup-guide.md:14]:
```suggestion
(delete line)
```

File: docs/guides/setup/es2k-setup-guide.md [hunk 8]
```diff
@@ -9,17 +9,27 @@ For the ACC, see [Building for the ES2K ACC](/guides/building-for-es2k-acc).
...
+module, networking-recipe.
```
Comment (ffoulkes) [inline docs/guides/setup/es2k-setup-guide.md:18]:
```suggestion
(delete line)
```

File: docs/guides/setup/es2k-setup-guide.md [hunk 9]
```diff
@@ -9,17 +9,27 @@ For the ACC, see [Building for the ES2K ACC](/guides/building-for-es2k-acc).
...
+
+Install the following dependencies for building networking-recipe.
```
Comment (ffoulkes) [inline docs/guides/setup/es2k-setup-guide.md:20]:
```suggestion
Install the following packages for building networking-recipe.
```

File: docs/guides/setup/dpdk-setup-guide.md [hunk 10]
```diff
@@ -5,17 +5,27 @@ for the DPDK target.
...
+
+Install the following dependencies for building networking-recipe.
```
Comment (ffoulkes) [inline docs/guides/setup/dpdk-setup-guide.md:16]:
```suggestion
Install the following packages for building networking-recipe.
```

**Review by ffoulkes — APPROVED** [review]
LGTM

I've suggested a number of edits that will reduce repetition and simplify the text.

**Review by n-sandeep — COMMENTED** [review]
LGTM

**Review by n-sandeep — APPROVED** [review]

**Pull request merged**